recipes/macsyfinder/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "MacSyFinder" %}
+{% set version = "2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c4b6a314c9185e6178b02e30d8e2da10380e0a9bdc9c2af78a80994fb083263a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7,<=3.10
+  run:
+    - python >=3.7,<=3.10
+    - colorlog
+    - PyYAML >=5.1.1
+    - packaging >=18.0
+    - networkx >=2.4
+    - pandas >=1.1.5,<=1.4.3
+    - colorama >=0.4.4
+    - certifi
+    - hmmer >=3.1b2,<=3.3.2
+
+test:
+  commands:
+    - macsyfinder --version
+
+about:
+  home: https://github.com/gem-pasteur/macsyfinder
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: GPL3
+  license_file: COPYING
+  summary: "MacSyFinder: Detection of macromolecular systems in protein datasets using systems modelling and similarity search"
+  doc_url: https://macsyfinder.readthedocs.io/en/latest
+
+
+extra:
+  recipe-maintainers:
+    - <put your github-id here>
+  doi:
+    - 10.1371/journal/pone.0110726
```

Comment:
does this tool already have a bio.tools ID?

Reply:
yes biotools:macsyfinder

Reply:
[6cdc25b](https://github.com/bioconda/bioconda-recipes/pull/36684/commits/6cdc25b3a456808215a1183633f6428dc4f89b92)